tests/Unit/Model/UrlTest.php

```diff
@@ -60,7 +60,7 @@ public function provideResolve()
         yield [
             'https://example.com',
             'https://example.com',
-            'https://example.com'
+            'https://example.com/'
```

Comment:
Is this a bug fix? (e.g. https://github.com/dantleech/fink/issues/96)

Reply:
Yes, I guess #96 can be closed after this PR is merged.